icons/unocss.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>UnoCSS</title><path d="M12.86 18.4a5.57 5.57 0 1 1 11.14 0 5.57 5.57 0 0 1-11.14 0ZM12.86 5.6A5.57 5.57 0 0 1 24 5.6v5.014c0 .307-.25.557-.557.557H13.417a.557.557 0 0 1-.557-.557V5.6ZM11.14 18.4A5.57 5.57 0 1 1 0 18.4v-5.014c0-.307.25-.557.557-.557h10.026c.307 0 .557.25.557.557V18.4Z"/></svg>
```

Comment:
I see some slight differences between the original icon and yours:
<img src="https://user-images.githubusercontent.com/2206639/175767094-5b630ba9-f884-4539-8b43-6fc22d2fcae6.svg" width="384" />

I believe they originate from the reduced path data precision. They are relatively subtle though. @sachinraja Do you think we can accept the SVG with those differences, or should we try to increase the precision?

Reply:
The path is not too big right now, so let's try increasing the precision

Reply:
Bumped precision from 3 to 4 in 4a1da3a.